Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -4838,7 +4850,7 @@ def main():  # pragma: no cover
             return_outputs(*get_machine_action_by_id_command(client, args))
 
         elif command == 'microsoft-atp-collect-investigation-package':
-            return_results(get_machine_investigation_package_command(client, args))
+            return_outputs(*get_machine_investigation_package_command(client, args))
```

Comment:
Why using the return_outputs and not the return results?

Reply:
that what was originally before my changes, so i just returned it as it was